python/hail/methods/impex.py

```diff
@@ -629,8 +630,124 @@ def get_vcf_metadata(path):
            contig_recoding=nullable(dictof(strlike, strlike)))
 def import_vcf(path, force=False, force_bgz=False, header_file=None, min_partitions=None,
                drop_samples=False, call_fields=[], reference_genome=None, contig_recoding=None):
-    return Env.hc().import_vcf(path, force, force_bgz, header_file, min_partitions,
-                               drop_samples, call_fields, reference_genome, contig_recoding).to_hail2()
+    """Import VCF file(s) as a matrix table.
+
+    Examples
+    --------
+
+    >>> ds = methods.import_vcf('data/example2.vcf.bgz')
+
+    Notes
+    -----
+
+    Hail is designed to be maximally compatible with files in the `VCF v4.2
+    spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`__.
+
+    :func:`.import_vcf` takes a list of VCF files to load. All files must have
+    the same header and the same set of samples in the same order (e.g., a
+    dataset split by chromosome). Files can be specified as :ref:`Hadoop glob
+    patterns <sec-hadoop-glob>`.
+
+
+    Ensure that the VCF file is correctly prepared for import: VCFs should
+    either be uncompressed (*.vcf*) or block compressed (*.vcf.bgz*). If you
+    have a large compressed VCF that ends in *.vcf.gz*, it is likely that the
+    file is actually block-compressed, and you should rename the file to
+    ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is
+    possible to import it to Hail using the ``force`` optional parameter.
+    However, this is not recommended -- all parsing will have to take place on
+    one node because gzip decompression is not parallelizable. In this case,
+    import will take significantly longer.
+
+
+    :func:`.import_vcf` does not perform deduplication - if the provided VCF(s)
+    contain multiple records with the same chrom, pos, ref, alt, all these
+    records will be imported as-is and will not be collapsed into a single
+    variant.
+
+    .. note::
+
+        Using the **FILTER** field:
+
+        The information in the FILTER field of a VCF is contained in the
+        ``filters`` row field. This annotation is a ``Set[String]`` and can be
+        queried for filter membership with expressions like
+        ``ds.filters.contains("VQSRTranche99.5...")``. Variants that are flagged
+        as "PASS" will have no filters applied; for these variants,
+        ``ds.filters.is_empty()`` is ``True``. Thus, filtering to PASS variants
+        can be done with :meth:`.MatrixTable.filter_rows` as follows:
+
+
+        >>> pass_ds = dataset.filter_rows(ds.filters.is_empty())
+
+
+    **Column Fields**
```

Comment:
Should we use

```
Column Fields
-------------------
```

here?

Reply:
will error out, napoleon only accepts known headers

Reply:
Ah, good to know!  I've been tempted before.